src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/rules/rule/index.tsx

```diff
@@ -14,47 +18,114 @@ type Props = {
   onEditRule: () => void;
   onDeleteRule: () => void;
   disabled: boolean;
+  listeners: DraggableSyntheticListeners;
+  rootStyle?: React.CSSProperties;
+  grabStyle?: React.CSSProperties;
+  forwardRef?: React.Ref<HTMLDivElement>;
 };
 
-function Rule({rule, onEditRule, onDeleteRule, disabled}: Props) {
+function Rule({
+  rule,
+  onEditRule,
+  onDeleteRule,
+  disabled,
+  listeners,
+  rootStyle,
+  forwardRef,
+  grabStyle,
+}: Props) {
   const {type, condition, sampleRate} = rule;
+
   return (
-    <React.Fragment>
-      <Column>
-        <StyledIconGrabbable disabled={disabled} />
-      </Column>
-      <Column>
-        <Type type={type} />
-      </Column>
-      <Column>
-        <Conditions condition={condition} />
-      </Column>
-      <CenteredColumn>
-        <SampleRate sampleRate={sampleRate} />
-      </CenteredColumn>
-      <Column>
-        <Actions
-          onEditRule={onEditRule}
-          onDeleteRule={onDeleteRule}
-          disabled={disabled}
-        />
-      </Column>
-    </React.Fragment>
+    <Wrapper ref={forwardRef} style={rootStyle}>
+      <Columns>
+        <Column>
+          <IconGrabbableWrapper {...listeners} disabled={disabled} style={grabStyle}>
+            <IconGrabbable />
+          </IconGrabbableWrapper>
+        </Column>
+        <Column>
+          <Type type={type} />
+        </Column>
+        <Column>
+          <Conditions condition={condition} />
+        </Column>
+        <CenteredColumn>
+          <SampleRate sampleRate={sampleRate} />
+        </CenteredColumn>
+        <Column>
+          <Actions
+            onEditRule={onEditRule}
+            onDeleteRule={onDeleteRule}
+            disabled={disabled}
+          />
+        </Column>
+      </Columns>
+    </Wrapper>
   );
 }
 
 export default Rule;
 
+const boxShadowBorder = '0 0 0 calc(1px / var(--scale-x, 1)) rgba(63, 63, 68, 0.05)';
+const boxShadowCommon = '0 1px calc(3px / var(--scale-x, 1)) 0 rgba(34, 33, 81, 0.15)';
+const boxShadow = `${boxShadowBorder}, ${boxShadowCommon}`;
+
+const Wrapper = styled('div')`
+  transform: translate3d(var(--translate-x, 0), var(--translate-y, 0), 0)
+    scaleX(var(--scale-x, 1)) scaleY(var(--scale-y, 1));
+  transform-origin: 0 0;
+  touch-action: manipulation;
+  --box-shadow: ${boxShadow};
+  --box-shadow-picked-up: ${boxShadowBorder}, -1px 0 15px 0 rgba(34, 33, 81, 0.01),
+    0px 15px 15px 0 rgba(34, 33, 81, 0.25);
+`;
```

Comment:
We would need these styles on other draggable items. Should this wrapper and other drag related styles be defined in `SortableItem`?

Reply:
sure thing, I will update that  😉